modules/nodes-pods-autoscaling-custom-prometheus-config.adoc

```diff
@@ -0,0 +1,199 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-autoscaling-custom.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-pods-autoscaling-custom-prometheus-config_{context}"]
+= Configuring the custom metrics autoscaler to use Prometheus Cluster Monitoring
+
+You can use the installed Prometheus Cluster Monitoring as a source for metrics for the custom metric autoscaler. However, there is some additional configuration you must perform.
```

Comment:
```suggestion
You can use the installed OpenShift Container Platform monitoring as a source for metrics for the custom metric autoscaler. However, there is some additional configuration you must perform.
```

Reply:
@zroubalik I added this to the topic also. Is this correct? We don't list this server address in the docs. I assume it's not needed for regular use.

> To use {product-title} monitoring as the soure, in the trigger/scaler, specify the `prometheus` type and use `https://thanos-querier.openshift-monitoring.svc.cluster.local:9092` as the `serverAddress`. `

Reply:
As of today the address needs to be specified, should be fixed in future release though.